src/styles/blocks/calendar-day.scss

```diff
@@ -0,0 +1,25 @@
+.day {
```

Comment:
you don't have this class, so you just duplicate your elements' styles. you don't need this file

Reply:
Got it. Thank you a lot ;)

Reply:
haven't fixed